backend/src/layer/data/database/factory.rs

```diff
@@ -30,6 +30,9 @@ impl DataAccessFactory {
     /// If wished, database migrations can be applied to create the wanted relations.
     /// # Errors
     /// if a migrations should, but could not run
+    ///
+    /// # Panics
+    /// if the connection to the database could not be established
```

Comment:
why do we expect here anyways? We probably can just use ? on an appropriate Error